lib/ramble/ramble/application.py

```diff
@@ -1138,6 +1138,14 @@ def _define_formatted_executables(self):
         based on the formatting requested.
         """
 
+        unformatted_str = ""
+        for cmd in self._command_list:
+            if unformatted_str:
```

Comment:
How about accumulating `cmd` in a list, and then `"\n".join(cmds)` at the end? (similarly for the formatted cmd below)

Reply:
Done! :) Thanks